### PR TITLE
Docker - Run orion in multi tenant

### PR DESCRIPTION
**Pull request by jmcanterafonseca**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -118,5 +118,5 @@ RUN \
     build-locale-archive
 
 WORKDIR /
-ENTRYPOINT ["/usr/bin/contextBroker","-fg"]
-EXPOSE 1026
\ No newline at end of file
+ENTRYPOINT ["/usr/bin/contextBroker","-fg", "-multiservice"]
```
Comment (fgalan) [inline docker/Dockerfile]: Maybe the fact that Orions runs in multiserve configuration should be mentioned in the Docker README.md? As a new bullet in the "A few points to consider" list.

Comment (jmcanterafonseca) [inline docker/Dockerfile]: good idea, fixed

**Comment by fgalan** [comment]
LGTM, but I think the LGTM from @LeandroGuillen (as main contributor to docker stuff in the repository) would be valuable. I'm assigning the PR to him.

File: docker/README.md [hunk 2]
```diff
@@ -83,7 +83,9 @@ As a result of this command, there is a context broker listening on port 1026 on
...
-* In `-p 1026:1026` the first value represents the port to listen in on localhost. If you wanted to run a second context broker on your machine you should change this value to something else, for example `-p 1027:1026`.
+* `--link mongodb:mongodb` assumes there is a docker container running a MongoDB image in your system,
+which name is `mongodb`. In case you need one type `sudo docker run --name mongodb -d mongo:2.6`.
+* In `-p 1026:1026` the first value represents the port to listen on localhost. If you want to run a second context broker
+on your machine you should change this value to something else, for example `-p 1027:1026`.
 * Anything after the name of the container image (in this case `orion`) is interpreted as a parameter for the Orion Context Broker. In this case we are telling the broker where the MongoDB host is, represented by the name of our other MongoDB container. Take a look at the [documentation](https://github.com/telefonicaid/fiware-orion) for other command-line options.
-
+* Orion will be running on multi-tenant mode
```
Comment (fgalan) [inline docker/README.md]: Maybe the "multit-tenant mode" could link to http://fiware-orion.readthedocs.org/en/develop/user/multitenancy/index.html (useful for people landing at dockerhub documentation without knowing what the multi-tenant mode is).

File: docker/Dockerfile (hunk 1 above)
Comment (LeandroGuillen) [inline docker/Dockerfile]: Question: wasn't multiservice enabled by default on later versions of Orion? Or maybe I just heard about it

Comment (fgalan) [inline docker/Dockerfile]: It wasn't enabled by default in the RPM configuration, but that changed recenctly (don't remember if in version 0.23.0 or 0.22.0). It think it was requested by @dmoranj , maybe he can provide additional information about the rationale of the change.

**Comment by jmcanterafonseca** [comment]
@fgalan @LeandroGuillen comments addressed

File: docker/README.md (hunk 2 above)
Comment (jmcanterafonseca) [inline docker/README.md]: fixed

**Comment by fgalan** [comment]
LGTM

File: docker/README.md [hunk 3]
```diff
@@ -83,7 +83,9 @@ As a result of this command, there is a context broker listening on port 1026 on
 A few points to consider:
 
 * The name `orion1` can be anything and doesn't have to be related to the name given to the docker image in the previous section.
-* `--link mongodb:mongodb` assumes there is a docker container running a MongoDB image in your system, whose name is `mongodb`. In case you need one type `sudo docker run --name mongodb -d mongo:2.6`.
-* In `-p 1026:1026` the first value represents the port to listen in on localhost. If you wanted to run a second context broker on your machine you should change this value to something else, for example `-p 1027:1026`.
+* `--link mongodb:mongodb` assumes there is a docker container running a MongoDB image in your system,
+which name is `mongodb`. In case you need one type `sudo docker run --name mongodb -d mongo:2.6`.
```
Comment (LeandroGuillen) [inline docker/README.md]: "whose" is correct, not "which"

Comment (jmcanterafonseca) [inline docker/README.md]: You are partially right, you can use 'whose', but according to 

http://dictionary.cambridge.org/grammar/british-grammar/relative-pronouns

whose is for people and animals usually; sometimes for things in formal situations
which is for animals and things

thus which is correct but as you were the original author I'm keeping 'whose' as I want the change to land as soon as possible :)

Comment (jmcanterafonseca) [inline docker/README.md]: fixed. @LeandroGuillen can you check?

thanks

**Comment by LeandroGuillen** [comment]
LGTM

**Pull request merged**